### ISSUE TITLE
Magic or not?

### ISSUE THREAD

**Issue by kestal** [kestal/univers#1]
If yes, what sort of ?

**Comment by kestal** [comment]
Possibilities include:
- _wh40k_ style (warp)
- _The black company_ style 
- _Tolkien_ style
- ??

I currently hesitate between something really free (à la black company) or something more restrictive.
I think the concept of warp is satisfying.

**Comment by lord-koko** [comment]
I would be for magic, dont know what kind yet though

**Comment by fahriel** [comment]
Is magic going to be an external or internal source? Does it come from the people who do it, or does it come from the world? Consequences: for external there would be magical places in the world, magical beasts. When magic is used it would have an effect on the world (its power source). For internal; the magic would be drawn from the life force of the user, some people may have a higher affinity for it than others (its in the bloodline). It would depend on the force of 'will' of the user.

**Comment by lord-koko** [comment]
I like the idea of an external source. 
No "real" reasons, but interesting stuff could be made out of it.
For example, wizard that travel to get/use/explore magical stuff.
Another one, is the use of elves. If elves keep this status of magical creature, it explains why the magic gets stronger when theyre a lot together. Maybe it becomes too strong ? cataclysmic ?
They also will be perfect travel companion/friend/slave of wizards.
A problem in this is that, if elves are considered as magical creature, it means that they are a source of magic, and in the external way of thinking magic, it would somehow objectify them and take out the subject out of them ?
Do we want to make elves objects ? or can we counter it simply by saying they are some specific kind of thing/people ? Which kind ?

I'm not sure to like the internal thing so much. The bloodline for example creates problematics that I dont want to explore, in relation to magic. Purity of the blood, racist wizard ?
A thing that I like in internal magic, is the concept that it is drained from your life energy. Throwing a fireball hurts you.
It is a bit less fun though, limitating what a wizard could do, to his physicall capacities.

Im sorry i didnt really answer anything, but just pointed out other things.

**Comment by ghost** [comment]
As it took too much space, I transfered this post in my fork located in the branch called **[patch-1](https://github.com/Adkorh/univers/tree/patch-1/Magic)**. This is divided into three files.

I think we can continue to debate it here.

**Comment by kestal** [comment]
OK, thanks for this long proposal, @Adkorh.

**Comment by ghost** [comment]
As I read a few things about the _warp_ of Warhammer 40k, I just wanted to point out that my concept of _praeter_ wasn't quite the same.
The warp is defined as the seat of the forces and gods of Chaos. The praeter wouldn't be like this. It would be just an undefined dimension with no preference to good or evil and with no link to gods or whatever.

I think there will be some disagreements about the [Schools of Magic](https://github.com/Adkorh/univers/blob/patch-1/Magic/3.%20The%20Frame%20of%20Magic.md) (called frame of magic now) ideas which bring the detailed guideline to imagine a new spell. That would be a nice subject to discuss.
I am also questioning myself about an idea of _magic comes with a price_, as there is actually no such thing in my proposal. What do you think about it ?

I also take the opportunity to ask the question: considering my proposal, would the _praeter_ and the god(s) be bound in any ways ? I personally think they shouldn't.

**Comment by kestal** [comment]
Yes, our notion of *warp* is different than W40K's in the sense than in 40K, warp is pretty much inherently evil, but our warp (at least in my view) is neutral.

Concerning the "Schools of Magic", I haven't read about them much, but I wasn't favorable to a too strict hierarchization of magic. We'll have to discuss it with @lord-koko and @fahriel.
*magic comes with a price* can lead to neat concepts, so if done well, I'd like this idea to be taken further.

Considering gods, there are none in the world. I'm pretty strongly decided on this.

**Comment by ghost** [comment]
Ok maybe the Schools concept is to straight. But more generally, are you still favorable to the rules that define the way magic comes into the world, those I wrote ? (And the generic/specific sources of power ?)
**Edit**: I changed the _generic/specific sources of power_ concept with this one instead: first each kind of spells need specific trainings to master, and secondly each kind of magic requires a specific environment to be performed. But all sources allow any kind of magic.

I brought some modifications to the Schools of Magic file, it is now renamed [Frame of Magic](https://github.com/Adkorh/univers/blob/patch-1/Magic/3.%20The%20Frame%20of%20Magic.md).

**Comment by kestal** [comment]
Haven't taken the time to read it all, yet. You're too quick for me !

**Comment by ghost** [comment]
Just go at your own pace I don't expect any quick answer.

I got ideas about two different ways magic could be seen. I will put these ideas in the form of excerpts of two books. You can see it in the [fork](https://github.com/Adkorh/univers/tree/patch-1/Magic).

Writing some stuff, I came to a question about the dwarves: do they have an aversion for magic (as they mainly do in the literature) ?

**Comment by kestal** [comment]
Not necessarily. We haven't yet made a decision in either direction.
In a sense, it could be interesting to have them averted to magic, or simply not sensitive to magic. But this could be too cliché and sort of the easy choice.
We could also make them actually practice magic, but in a very limited/directed form
